### PR TITLE
[json-rpc] Add related events to currency info

### PR DESCRIPTION
**Pull request by runtian-zhou**
## Motivation

Add events in json-rpc response for get_currency_info.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Not yet.

### CODE REVIEW

**Comment by runtian-zhou** [comment]
@christinacelee @thefallentree  do you have any suggestions on how we should test the json-rpc?

File: json-rpc/types/src/views.rs [hunk 1]
```diff
@@ -499,6 +499,11 @@ pub struct CurrencyInfoView {
     pub scaling_factor: u64,
     pub fractional_part: u64,
     pub to_lbr_exchange_rate: f32,
+    pub mint_events: BytesView,
```
Comment (thefallentree) [inline json-rpc/types/src/views.rs:502]: we should add _key as suffix here

**Comment by thefallentree** [comment]
in general, you just modify https://github.com/libra/libra/blob/master/json-rpc/src/tests/unit_tests.rs#L50 to verify the event key matches what you expected. 

Might also need to change mockdb() to record that during intitilization as well.

**Comment by runtian-zhou** [comment]
I'm not sure how to change the mock_db for testing. There isn't a testing for `get_currencies` right now. It seems to me that we just generate random fake blocks in the mockdb and thus there's actually no currency info stored in the mock_db right now.

**Comment by runtian-zhou** [comment]
Just tested the impl locally by:
1. Running a local libra swarm
2. Query the currency via curl:
```
curl http://localhost:65028  -H 'Content-Type: application/json' -X POST  --data '{"jsonrpc":"2.0","method":"get_currencies","params":[],"id":1}'
{"id":1,"jsonrpc":"2.0","result":[{"burn_events_key":"02000000000000000000000000000000000000000a550c18","cancel_burn_events_key":"04000000000000000000000000000000000000000a550c18","code":"Coin1","exchange_rate_update_events_key":"05000000000000000000000000000000000000000a550c18","fractional_part":100,"mint_events_key":"01000000000000000000000000000000000000000a550c18","preburn_events_key":"03000000000000000000000000000000000000000a550c18","scaling_factor":1000000,"to_lbr_exchange_rate":0.5},{"burn_events_key":"07000000000000000000000000000000000000000a550c18","cancel_burn_events_key":"09000000000000000000000000000000000000000a550c18","code":"Coin2","exchange_rate_update_events_key":"0a000000000000000000000000000000000000000a550c18","fractional_part":100,"mint_events_key":"06000000000000000000000000000000000000000a550c18","preburn_events_key":"08000000000000000000000000000000000000000a550c18","scaling_factor":1000000,"to_lbr_exchange_rate":0.5},{"burn_events_key":"0c000000000000000000000000000000000000000a550c18","cancel_burn_events_key":"0e000000000000000000000000000000000000000a550c18","code":"LBR","exchange_rate_update_events_key":"0f000000000000000000000000000000000000000a550c18","fractional_part":1000,"mint_events_key":"0b000000000000000000000000000000000000000a550c18","preburn_events_key":"0d000000000000000000000000000000000000000a550c18","scaling_factor":1000000,"to_lbr_exchange_rate":1.0}]}
```

**Pull request closed without merging**